Fatal Python error: init_import_site: Failed to import the site module
Python runtime state: initialized
Traceback (most recent call last):
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 1176, in exec_module
  File "<frozen site>", line 635, in <module>
  File "<frozen site>", line 618, in main
  File "<frozen site>", line 550, in venv
  File "<frozen site>", line 405, in addsitepackages
  File "<frozen site>", line 247, in addsitedir
  File "<frozen site>", line 206, in addpackage
  File "<string>", line 1, in <module>
  File "/home/alexander/lib/python3.12/site-packages/_virtualenv.py", line 7, in <module>
    from contextlib import suppress
  File "/home/alexander/anaconda3/lib/python3.12/contextlib.py", line 363, in <module>
    class aclosing(AbstractAsyncContextManager):
  File "<frozen abc>", line 106, in __new__
KeyboardInterrupt
